### DISCUSSION TITLE
Can I be a User of this subnet ?

### DISCUSSION THREAD

**Discussion by nathanlebeau** [opentensor/prompting#430] [Q&A]
Hello I was wondering if this subnet support any user feature for the moment ? Thanks.

**Comment by mitselek** [comment]
## Champollion — CRD.EE, PCS.EE, EA Sparx

### CRD.EE User Manual (208994326)

**What I found:**

CRD.EE is a production Oracle schema (`crd`) plus APEX UI, already live. It holds two categories of data:
1. **RNE reference data** — Countries, Companies, Primary locations, Subsidiary locations, Subsidiary location types — replicated nightly from RNE SOAP API (full, non-destructive, Baltic locations only)
2. **TAF/TAP message enumerations** — message types and status codes, managed via schema migrations

Other apps consume it by direct schema grant (`grant select, references on crd.message_types to pcs`). Architecture diagram is at the page. Manual import also possible via CRD GitLab repo CLI tools.

Permission model is simple: single role `CRD_USAGE_RO` for all view access. UI is read-only (5 list views matching the 5 RNE data types, with a VJS schema comparison view where applicable).

**Why it matters:**

Our ADR-0008 lists CRD.EE as a shared target database. This confirms it is **already built and in production** as of 2024-09. The RNE replication being nightly/non-destructive/Baltic-scoped is new detail not in our specs — this constrains what VJS migration apps can assume about CRD freshness and coverage. The `vjs` schema comparison view means CRD.EE already has data quality logic we should understand before migrating apps that touch location or company data (e.g. f203, f301).

**Recommend extracting:** Yes — the architecture diagram and permission table are spec-quality material. Should feed into ADR-0008 and the specs for any cluster touching path requests or reference data.

---

### PCS.EE User Manual (201195531)

**What I found:**

PCS.EE is **application 301** (APEX app f301 in our inventory, the one with the open PO question in Discussion #29). It is a production Path Coordination System with two user roles:
- **IM side** (`PCS_MONITOR_AS_IM`, `PCS_PROCESS_AS_IM`) — handles received path requests, creates PathNotAvailable messages
- **RU side** (`PCS_MONITOR_AS_RU`, `PCS_PROCESS_AS_RU`, `PCS_CREATE_AS_RU`) — creates and manages path requests on behalf of Railway Undertakings

Page inventory: 12 APEX pages (pages 5, 6, 8, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23). Named pages confirm the LDZ interop: "Accept/reject path offers from the IM (LDZ)" is an explicit step in the RU flow.

Attachments: 4 Word user manual docs (handling a received PR, creating a PR, managing a sent PR, received path cancelations). Last updated 2024-10-01 (version 40).

**Why it matters:**

This directly resolves the open question from Discussion #29: f301 is **in production** handling real TAF/TAP path coordination with LDZ. The permission model (`PCS_MONITOR_AS_IM` etc.) is the actual target auth model for the migrated version — these named permissions should flow into our auth-model spec. Our inventory for f301 currently has no page-level detail; this page gives us the complete page list.

**Recommend extracting:** Yes, high priority — resolves the Discussion #29 open question, and the permission table + page inventory are directly usable in the f301 spec.

---

### EA Sparx (543621122)

**What I found:**

Short operational guide (version 1, created 2024-09-24). Key facts:
- The EA repository is **file-based** (`.qeax`), not Oracle-backed — stored in GitLab at `evr/enterprise-architect-documentation`, file `LATEST-evr-master.qeax`
- Solita holds floating EA Sparx licences
- Import path for DB objects: Oracle Instant Client 19.24 → ODBC datasource → EA Sparx Develop → Data Modeling → Import

**Why it matters:**

This confirms our EA model source (`ER_VJS_v0.8.0.qeax`) is the same format and toolchain as the production Solita/EVR model. The GitLab repo `evr/enterprise-architect-documentation` is separate from the VJS apps repo — it is the live EA model we should compare against. Our extraction used the same `.qeax` SQLite approach, so the toolchain is validated. No new spec content here, but it confirms the EA model location for future reference.

**Recommend extracting:** No — this is a tooling guide, not domain content. The GitLab repo path is worth recording as a reference memory.

(*AR:Champollion*)
